robottelo/config/validators.py

```diff
@@ -161,6 +155,12 @@
             'http_proxy.password',
             must_exist=True,
         ),
+        # validate http_proxy_ipv6_url only if server.is_ipv6 is True
+        Validator(
+            'http_proxy.http_proxy_ipv6_url',
```

Comment:
I think keeping the global HTTP Proxy for all satellite applications, the server conf is a good place.

HTTP Proxy conf is made to configure within the satellite, whereas IPV6 HTTP proxy is inside and outside as well to have cross network communication.

Reply:
`HTTP Proxy conf is made to configure within the satellite` It's not, it can be used to configure on system level as well. It's not different from Ipv6 HTTP proxy. It can do things ipv6 can do. so I don't see a reason to differentiate. https://github.com/SatelliteQE/robottelo/pull/16928#discussion_r1844266242

Reply:
Another reason keeping it in `server.yaml` is that's the yaml sets global things for robottelo including the `is_ipv6` switch. Though its duplicating the key from http_proxy yaml, having it under `server.yaml` and near to `is_ipv6` setting increases the usability of robottelo.

Reply:
By that logic, we'll be making Robottelo unnecessarily bulky and a lot of other settings could fall into such reasoning. Not to mention that we'll be basically ignoring the very general rule of clustering similar settings in same yaml. But I'd leave the final decision to you/JPL.